manifests/init.pp

```diff
@@ -185,6 +185,12 @@
 #
 # $dhcp_pxeserver::             DHCP "next-server" value, defaults otherwise to IP of dhcp_interface
 #
+# $dhcp_pxefilename::           DHCP "filename" value, defaults otherwise to pxelinux.0
+#
+# $dhcp_network::               DHCP server network value, defaults otherwise to undef
```

Comment:
Could you add that it will determine the network based on `dhcp_interface` when it's undef? Same for netmask.